eeps/eep-0049.md

```diff
@@ -204,23 +224,23 @@ as:
 
     pre_check(Action, User, Context, ExternalThingy) ->
         begin
-            _ <~ check_request(Context, User),
-            _ <~ check_permissions(Action, User),
-            _ <~ check_dispatch_target(ExternalThingy),
-            _ <~ check_condition(Action, Context),
+            ok <- check_request(Context, User),
+            ok <- check_permissions(Action, User),
+            ok <- check_dispatch_target(ExternalThingy),
+            ok <- check_condition(Action, Context),
             dispatch(Action, User, Context)
-        end
+        end.
```

Comment:
Nitpick: I notice you added a dot here, perhaps add them everywhere.

Reply:
they're generally there when a function is used, omitted when only the expression is shown. Had a few stragglers that I'll fix up though.